.github/workflows/scripts/install.sh

```diff
@@ -33,19 +33,10 @@ if grep -i bionic /etc/os-release ; then
   sudo apt-add-repository --yes --update ppa:ansible/ansible
 fi
 
-# Updating ansible past 2.9.6 (to 2.9.8+) is necessary to manage Fedora 32:
-# https://github.com/ansible/ansible/pull/68211
-# And to 2.9.16 for a bug with managing systemd
-# https://github.com/ansible/ansible/issues/71528#issuecomment-729778048
-if grep -i focal /etc/os-release ; then
-  # The 21.04 version.
-  ANSIBLE_DEB=ansible_2.9.16+dfsg-1.1_all.deb
-  curl -L --output $ANSIBLE_DEB http://mirrors.kernel.org/ubuntu/pool/universe/a/ansible/$ANSIBLE_DEB
-  sudo apt install ./$ANSIBLE_DEB
-  sudo rm ./$ANSIBLE_DEB
-else
-  sudo apt install ansible
-fi
+sudo apt remove ansible
+pip3 install -U pip
+pip3 install "ansible>2.9,<3"
```

Comment:
it only worked on 2.10

Reply:
Per our meeting convo, if we only support 2.10, we should communicate that.

Also possibly bump the version required in our playbooks/ directory (search it for "ansible_version.full").

Reply:
the issue is:
I'm not sure if it is something related to virtualization or FIPS itself, I only know it was failing before

Reply:
I don't know what AI should I take, not every user will use FIPS, 2.10 is only for FIPS cases and I'm not totally sure if it is a FIPS thing or GHA virtualization issue

Reply:
Let's lean towards an approach that is low maintenance for us.

We could have only our fips tests test Ansible 2.10 and 3.0.

Or we could drop Ansible 2.9 entirely.

You could ask the mailing list about this, see if anyone would object to dropping 2.9.